src/include/daos/container.h

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2016-2023 Intel Corporation.
+ * (C) Copyright 2016-2024 Intel Corporation.
```

Comment:
erroneous copyright update after changes were backed out. Here and others

Reply:
I'll do another pass and fix these when I'm done with other changes.